docker/docker-compose.yml

```
@@ -68,6 +68,8 @@ services:
 
   # This "container" is a workaround to pre-create search indices
   elasticsearch-setup:
+    labels:
+      setup_job: true
```

Would be nice to qualify this with `datahub`

```suggestion
      datahub_setup_job: true
```